scripts/table_updates/update_table_schema.py

```diff
@@ -19,185 +16,356 @@ def copy_table_schema(syn, from_table_id, to_table_id):
     to_table_schema.columnIds = from_table_schema.columnIds
     return to_table_schema
 
+
 def create_synapse_column(name, col_type, max_size):
     """
-    Create a new Synapse Column with given info
+    Create a new Synapse Column with given info. ??? may extend, string. use data element column types
     """
-    if col_type in ['INTEGER','DOUBLE','LARGETEXT']:
-        new_column = Column(name=name,
-                            columnType=col_type)
+    if col_type in ["INTEGER", "DOUBLE", "LARGETEXT"]:
+        new_column = Column(name=name, columnType=col_type)
     else:
         max_size = 500 if pandas.isna(max_size) else int(max_size)
-        new_column = Column(name=name,
-                            columnType=col_type, 
-                            maximumSize=max_size)
+        new_column = Column(name=name, columnType=col_type, maximumSize=max_size)
     return new_column
 
+
 def _expand_checkbox_vars(row):
+    """Expand checkbox variables into multiple columns
```

Comment:
nit: could you expand on this docstring, e.g: 

```
Expand variables with type checkbox in the GENIE BPC No PHI Data Elements Catalog table into multiple columns

Args:
        row (pd.Series): A row of the Data Elements Catalog dataframe
```

Reply:
Yes, in progress.